src/pages/dmt5/index.js

```diff
@@ -1,6 +1,6 @@
 import React, { useState, useEffect } from 'react'
 import { graphql, useStaticQuery } from 'gatsby'
-import { WhyTrader, StartTrader, DownloadApp, Flexibility, DBanner } from './_lazy-load'
+import { WhyTrader, StartTrader, DownloadApp, Flexibility, DBanner, MarginCalculator /*SwapFreeTrading*/ } from './_lazy-load'
```

Comment:
```suggestion
import { WhyTrader, StartTrader, DownloadApp, Flexibility, DBanner, MarginCalculator } from './_lazy-load'
```
We can omit `SwapFreeTrading` to avoid confusion